### DISCUSSION TITLE
在Thinkphp中执行服务端报Failed to decode request contents.异常

### DISCUSSION THREAD

**Discussion by beijin1949** [w7corp/easywechat#2860] [Q&A]
本人代码：
        $config = [
            'app_id' => env("WX_APP_ID"),
            'secret' => env("WX_SECRET"),
            'token' => env("WX_TOKEN"),
            'aes_key'=> '',
        ];
        $app = new Application($config);
        $server = $app->getServer();

        $rep = $server->serve();
        $this->res($rep);
        
        实际测试只要执行$server->serve();就会报这个错误，并且无法通过服务端的token校验，直接打开页面会显示
#0 [0]BadRequestException in Message.php line 62
        Failed to decode request contents.
        这个错误

**Comment by xshenzhi** [comment]
关闭debug
debug=false
